ee/desktop/notify/notify_darwin.m

```diff
@@ -90,11 +90,6 @@ BOOL doSendNotification(UNUserNotificationCenter *center, NSString *title, NSStr
 BOOL sendNotification(char *cTitle, char *cBody, char *cActionUri) {
     UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
 
-    // To be removed later -- for troubleshooting purposes only
```

Comment:
@RebeccaMahany Is this worth keeping around at this point? I don't want to remove it if we think it could still be useful for an issue we've yet to resolve.

Reply:
Oh ha just commented on this -- yep this is fine to remove, thanks for getting it!